Vagrantfile

```diff
@@ -32,7 +33,7 @@ $vm_cpus = 1
 $shared_folders = {}
 $forwarded_ports = {}
 $subnet = "172.17.8"
-$os = "ubuntu"
+$os = "ubuntu1804"
```

Comment:
instead of hardcoding an ubuntu version, we might make SUPPORTED_OS mutable and then merge a ubuntu key with a predefined version as value